lint_test/proper_super_calls_test.dart

```diff
@@ -75,3 +75,7 @@ class MyClass {
   dispose() {}
   initState() {}
 }
+
+abstract interface class Disposable {
+  void dispose();
+}
```

Comment:
Could you please add a comment here explaining what this testing is?

Reply:
Yep, added)